.gitignore

```diff
@@ -26,3 +26,7 @@ public/*
 coverage
 lib
 artifacts
+
+tls.cert
```

Comment:
Why are these needed? Are they artefacts resulting from a 'normal' test run or specific to your dev workflow? If they were just a 1 off artifact I wouldn't add them.